vnext/template/shared-lib/proj/BuildFlags.props

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This was in the app template before.

This one is a bit special. Meant to allow customizing Microsoft.ReactNative source build. The project looks in solutionDir for this file, to change its own behavior.

Added to allow specialized compilation for Hermes, since we cannot yet enable its conditional compilation in MSRN by default.

Reply:
yep, pre-existing art :) we can revisit if need be. It's nice to have experimental features be gated by a file that's just that.

Reply:
https://github.com/microsoft/react-native-windows/pull/7075